src/cargo-kani/src/call_goto_instrument.rs

```diff
@@ -124,77 +121,3 @@ impl KaniSession {
         self.run_suppress(cmd)
     }
 }
-
-/// Collect all vtable restriction metadata together, and write one combined output in CBMC's format
-fn link_function_pointer_restrictions(
```

Comment:
Next time, I would prefer if you leave the refactoring of unrelated code to separate PR. Thanks